### ISSUE TITLE
search box keyboard input does not work under OS X

### ISSUE THREAD

**Issue by iandennismiller** [smathot/qnotero#14]
I've installed Qnotero under OS X using the following process:

```
mkvirtualenv qnotero
wget http://files.cogsci.nl/software/qnotero/qnotero_0.48.tar.gz
sudo mkdir /usr/share/qnotero && chown $USER /usr/share/qnotero
vi setup.py # comment out the .desktop icon
python setup.py install
qnotero
```

When I attempt to search, it appears that no keyboard input is recognized.  Control characters like esc, space, tab, or enter also do not work.

After launching, I can get to preferences, then use the chooser to find my Zotero library.  I am able to use the mouse to click to 'UI focus' the search box, and I can click away resulting in the search box losing focus.  When I click outside qnotero, the whole interface disappears.  I have a book icon in the menu bar that I can use to reveal qnotero again

In other words, it looks like most of the rest of the functionality is present on OS X. Great work!  Are there any suggestions for ensuring keyboard input is robust across platforms?
